### PR TITLE
MINIFICPP-692 - Ensure calls to get a flowfile are consistent across …

### PR DESCRIPTION
**Pull request by arpadboda**
…languages

Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [ ] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nanofi/include/api/nanofi.h [hunk 1]
```diff
@@ -233,8 +233,6 @@ flow_file_record *get_next_flow_file(nifi_instance *, flow *);
  **/
 size_t get_flow_files(nifi_instance * instance, flow * flow, flow_file_record ** flowfiles, size_t size);
 
-flow_file_record *get(nifi_instance *,flow *, processor_session *);
```
Comment (arpadboda) [inline nanofi/include/api/nanofi.h]: This was only used by Python API before (otherwise you can't get a session ptr)

**Comment by arpadboda** [comment]
Python and C API use the same func. to get flow file in custom processors.
Renamed this func. to "get" as requested in the corresponding Jira ticket. 
Extended attribute handling in Python API.

**Comment by phrocker** [comment]
Changes look good. Will run tests a little later and merge if all is good

**Comment by phrocker** [comment]
Changes look good from a cursory look, will run some tests and merge if all is good thanks!

**Pull request closed without merging**